### PR TITLE
Features/newsletter component

### PR DESCRIPTION
**Pull request by asraraAhmed**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,7 +14,9 @@
     "@angular/common": "^15.1.0",
     "@angular/compiler": "^15.1.0",
     "@angular/core": "^15.1.0",
+    "@angular/flex-layout": "^15.0.0-beta.42",
```
Comment (SarahFurqan) [inline package.json]: beta version should not be added

File: src/app/Components/MainDashbordComponents/footer/footer.component.css [hunk 2]
```diff
@@ -11,16 +11,16 @@ footer {
 
 .data1 {
```
Comment (SarahFurqan) [inline src/app/Components/MainDashbordComponents/footer/footer.component.css]: data1 , btn1 btn2 classes should be rename to some realistic classes name

File: src/app/Components/MainDashbordComponents/footer/footer.component.css [hunk 3]
```diff
@@ -11,16 +11,16 @@ footer {
...
+    color: var(--secondary-color);
 }
 
 .data1:hover {
-    color: #5EBE80 !important;
+    color: var(--primary-color) !important;
     cursor: pointer;
 }
 
 .ftr {
```
Comment (SarahFurqan) [inline src/app/Components/MainDashbordComponents/footer/footer.component.css]: change class name

File: src/app/Components/MainDashbordComponents/footer/footer.component.css [hunk 4]
```diff
@@ -38,24 +38,24 @@ footer {
 }
 
 .btn1:hover {
```
Comment (SarahFurqan) [inline src/app/Components/MainDashbordComponents/footer/footer.component.css]: primary-button-hover class should be in common css and called that class everywhere

File: src/app/Components/MainDashbordComponents/footer/footer.component.html [hunk 5]
```diff
@@ -105,21 +105,21 @@ <h5>
     </div>
 
     <div class="col md-4 lg-4">
-        <h5><span style="color: #29416F; font-family: outfit;">Contact Info</span></h5>
+        <h5><span style="color: var(--secondary-color); font-family: outfit;"> Contact Info</span></h5>
```
Comment (SarahFurqan) [inline src/app/Components/MainDashbordComponents/footer/footer.component.html:78]: Please remove inline style, instead use same class for primary-color class and template-font class from common

**Review by SarahFurqan — CHANGES_REQUESTED** [review]

**Comment by asraraAhmed** [comment]
changes done

**Pull request merged**